### PR TITLE
Implementing Styled Components

### PR DESCRIPTION
**Pull request by CFIALeronB**
Implementing Styled Components and removing unused CSS objects.

#7

### CODE REVIEW

**Review by k-allagbe — APPROVED** [review]
Don't forget to add more details to the issue.

**Review by rngadam — CHANGES_REQUESTED** [review]
provide more details on the why and how

File: src/_versions.ts [hunk 1]
```diff
@@ -9,10 +9,10 @@ export interface TsAppVersion {
     gitTag?: string;
 };
 export const versions: TsAppVersion = {
-    version: '0.1.1',
+    version: '0.1.0',
```
[the remark this answers is not in the document]
Comment (rngadam) [reply src/_versions.ts:12]: @redat97 this is what I was talking about, you have to make sure the package.json version is getting incremented to that the _version.ts is incremented.

File: src/pages/search/SearchPage.tsx [hunk 2]
```diff
@@ -26,16 +24,17 @@ const SearchPage: React.FC = () => {
   }, [searchTerm]);
 
   return (
-    <LayoutContainer>
```
Comment (rngadam) [inline src/pages/search/SearchPage.tsx]: I'm suspicious that we're replacing something readable that explains what the component is with styling parameters. Seems to be a step backward. Can you explain what is going on here?

**Review by rngadam — CHANGES_REQUESTED** [review]

File: src/pages/search/SearchPage.tsx (hunk 2 above)
Comment (CFIALeronB) [reply src/pages/search/SearchPage.tsx]: > I'm suspicious that we're replacing something readable that explains what the component is with styling parameters. Seems to be a step backward. Can you explain what is going on here?

Seems to be an outdated file your referencing. I merged Guy's branch and removed  div component tags and replaced them with Stylized Components:

   <div className={styles.layout}> <---- old
    <LayoutContainer> <--- New

**Review by rngadam — CHANGES_REQUESTED** [review]
looks like this needs to be rebased to fix conflicts with the main branch

**Review by rngadam — APPROVED** [review]

**Pull request merged**